cdap-ui/app/cdap/components/PipelineScheduler/index.js

```diff
@@ -39,8 +39,9 @@ import {Observable} from 'rxjs/Observable';
 import {PROFILES_DROPDOWN_DOM_CLASS} from 'components/PipelineScheduler/ProfilesForSchedule';
 import {MyScheduleApi} from 'api/schedule';
 import T from 'i18n-react';
-import {PROFILE_NAME_PREFERENCE_PROPERTY} from 'components/PipelineConfigurations/ConfigurationsContent/ComputeTabContent/ProfilesListView';
+import {PROFILE_NAME_PREFERENCE_PROPERTY, PROFILE_PROPERTIES_PREFERENCE} from  'components/PipelineDetails/ProfilesListView';
```

Comment:
nit: extra space after `from`. This is happening at several imports of `components/PipelineDetails/ProfilesListView`